<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QMainWindow" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel#loginImage{
	image:url(:/resources/image/logo.jpg);
	height:440px;
	width:440px;
}

QLabel#loginTitle{
	image:url(:/resources/image/loginTitle.png);
	height:90px;
}

QToolButton#loginMusic{
	border-image:url(:/resources/image/musicYes.png);
}

QToolButton#loginIntroduction{
	border-image:url(:/resources/image/loginIntroduction.png);
}
QToolButton#loginIntroduction:hover{
	border-image:url(:/resources/image/loginIntroduction1.png);
}

QToolButton#loginPractice{
	border-image:url(:/resources/image/loginPractice.png);
}
QToolButton#loginPractice:hover{
	border-image:url(:/resources/image/loginPractice1.png);
}

QToolButton#loginHuman{
	border-image:url(:/resources/image/loginHuman.png);
}
QToolButton#loginHuman:hover{
	border-image:url(:/resources/image/loginHuman1.png);
}


</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="loginImage">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>140</y>
      <width>420</width>
      <height>420</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="loginTitle">
    <property name="geometry">
     <rect>
      <x>16</x>
      <y>19</y>
      <width>600</width>
      <height>88</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QToolButton" name="loginIntroduction">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>260</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonIconOnly</enum>
    </property>
   </widget>
   <widget class="QToolButton" name="loginPractice">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>370</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QToolButton" name="loginHuman">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>480</y>
      <width>200</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QToolButton" name="loginMusic">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>50</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
